src/uucore/src/lib/features/signals.rs

```diff
@@ -196,6 +200,36 @@ pub fn signal_name_by_value(signal_value: usize) -> Option<&'static str> {
     ALL_SIGNALS.get(signal_value).copied()
 }
 
+#[cfg(unix)]
+pub fn enable_pipe_errors() -> Result<()> {
+    // SAFETY: this function is safe as long as we do not use a custom SigHandler -- we use the default one.
+    match unsafe { signal(SIGPIPE, SigDfl) } {
+        Ok(_) => Ok(()),
+        _ => Err(Error::from(ErrorKind::Other)),
+    }
+}
+
+#[cfg(not(unix))]
+pub fn enable_pipe_errors() -> Result<()> {
+    // Do nothing.
+    Ok(())
+}
```

Comment:
Maybe we should not include these and add `#[cfg(unix)]` at the call sites? What do you think?

Reply:
Hmm, that seems reasonable.